### DISCUSSION TITLE
Strange Issue with syx on Pi4 and fail to build on Pi2

### DISCUSSION THREAD

**Discussion by DJTheDarksider** [probonopd/MiniDexed#237] [Q&A]
First of all MANY MANY Thanks for creating this awesome project!

I have a strange issue - I tried to build minidexed with a RasPi2, but didn´t sucseed (Display just showed white Sqares and no sound via HDMI Audio (Adapter connected to a Monitor but this also didn´t show anything) 

Gladly I also had a Pi4 where I got the Display and Rotary Encoder to work as expected...
BUT
with the audio changed to pwm I got sound but it was quite Blurry so I deactivated the Reverb, so it was clear that different Patches played at the same time - changed the Midi channel settings from 255 to the actual chan number and was able to get the different sounds to work when using a Midi to USB Cable and driving it from Ableton (No luck with a Novation SL49 MK3 - this one played on all channels via USB even with selecting single channels) 

Now to my actual issue.
I use a bank I created in the VST Version of Dexed named as sugested (which works fine - I use the same bank on my Volca FM, which isn´t multitimbral or Velocity sensitive unfortunately)
Some of the patches stay mute for some  strange reason.
Is Minidexed based on the actual "Dexed" by DigitalSuburban (in which case it should be patch compatible) 
I attached the Bank (zipped), maybe somebody is able to test if the patches (For instance STRM1 LNGA, BELLS or or Spaceghost didn´t work)
Any help is appreciated (either on the issue with the Pi2 or the Syx one) 
Thanks in advance.

[000000_Dexed_Darksider.zip](https://github.com/probonopd/MiniDexed/files/8718614/000000_Dexed_Darksider.zip)

**Comment by probonopd** [comment]
Welcome @DJTheDarksider. MiniDexed should work on Raspberry Pi 2, but depending on how much you are trying to do at the same time the additional horsepower of the Raspberry Pi 4 is better.

You might be running into issue https://github.com/probonopd/MiniDexed/issues/83. Maybe you can look into it and find out what is going wrong there? Do the non-working voices all have something in common?

MiniDexed uses [Synth_Dexed](https://codeberg.org/dcoredump/Synth_Dexed/) which itself is based on Dexed which is in turn based on MSFA.

**Reply by dcoredump** [reply]
I think I have to investigate this. So many things to do....
